backend/libraries/types/src/chit.rs

```diff
@@ -15,3 +15,9 @@ pub enum ChitEarnedReason {
     DailyClaim,
     Achievement(String),
 }
+
+#[derive(CandidType, Serialize, Deserialize, Clone, Debug)]
+pub struct ChitUserBalance {
+    pub user_id: UserId,
+    pub balance: i32,
```

Comment:
Maybe this should be a u32? And we exclude negatives.

Reply:
Otherwise if the first user in a season gets a deduction, they'll still appear top.